Azure/HelloCloud.Headless/Package.appxmanifest

```diff
@@ -0,0 +1,36 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" xmlns:iot="http://schemas.microsoft.com/appx/manifest/iot/windows10" IgnorableNamespaces="uap mp iot">
+  <Identity Name="HelloCloud-uwp" Publisher="CN=arturl" Version="1.0.0.0" />
+  <mp:PhoneIdentity PhoneProductId="c413c740-710c-4065-b251-9291b6aa05c9" PhonePublisherId="00000000-0000-0000-0000-000000000000" />
+  <Properties>
+    <DisplayName>HelloCloud</DisplayName>
+    <PublisherDisplayName>arturl</PublisherDisplayName>
```

Comment:
also copy the pfx file from Blinky and rename it to HelloCloud_TemporaryKey.pfx (that enusres that all of our samples have the same pfx that matches publisher MSFT and have the same expiration date)